kafka-connect-s3/src/test/java/io/confluent/connect/s3/S3SinkTaskTest.java

```diff
@@ -100,6 +102,30 @@ public void testWriteRecord() throws Exception {
     verify(sinkRecords, validOffsets);
   }
 
+  @Test
+  public void testWriteTwoRecords() throws Exception {
```

Comment:
might need to rename this test. I think the point here was to test if the task can survive a null valued record? so maybe `testPutNullRecord`, and maybe we can send maybe 5-6 records after that and check the records that were sent before and after the null were successfully received at the (mock) destination? you can use `createRecords` to generate some sample records.